<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManageUsers</class>
 <widget class="QWidget" name="ManageUsers">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>771</width>
    <height>557</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ManageUsers</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLineEdit, QPlainTextEdit, QComboBox, QRadioButton, QPushButton, QDateTimeEdit
{
font: 15px &quot;Times New Roman&quot;;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="titleLabel">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Manage user accounts</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="infoLabel">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Press delete button to delete an account.</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="searchUserLabel">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Search for users</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="searchUserInput"/>
     </item>
     <item>
      <widget class="QTableWidget" name="userTableWidget">
       <property name="styleSheet">
        <string notr="true">QTableWidget
{
	font: 15px &quot;Times New Roman&quot;;
}

QHeaderView::section 
{
    background-color: silver;
    border: none;
	font-family: Times new Roman;
	font-size: 16px;
}

QTableWidget::item:selected
{ 
background-color: rgb(36, 36, 36);
color: white;
}</string>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="gridStyle">
        <enum>Qt::DashLine</enum>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="addUsersFromTxtButton">
       <property name="text">
        <string>Add users from txt file</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
